src/client/sky.cpp

```diff
@@ -766,3 +624,105 @@ void Sky::update(float time_of_day, float time_brightness,
 			video::SColorf(pointcolor), m_horizon_blend() * 0.25);
 	}
 }
+
+void Sky::draw_sun(video::IVideoDriver *driver, float sunsize, video::SColor suncolor,
+		video::SColor suncolor2, float wicked_time_of_day)
```

Comment:
Indentation of function declarations is always 1 tab, see code style guidelines https://dev.minetest.net/Code_style_guidelines

Reply:
Thanks for the link.